src/clients/ProfitClient.ts

```diff
@@ -352,7 +377,7 @@ export class ProfitClient {
     }
   }
 
-  private async updateGasCosts(): Promise<void> {
+  async updateGasCosts(): Promise<void> {
```

Comment:
updateGasCost() isn't called externally anymore, so we can probably revert this change.

```suggestion
  private async updateGasCosts(): Promise<void> {
```